src/table/table.scss

```diff
@@ -47,8 +47,8 @@
 
   .iui-cell:not(.iui-slot) {
     @include iui-focus;
-    flex-wrap: wrap;
     column-gap: $iui-xs;
+    padding-block: $iui-baseline * 0.5;
```

Comment:
<90% support is enough?

Reply:
Idk, most of the browsers have this since last year, some of them even since 2020.